Examples/Detectors/TelescopeDetector/include/ActsExamples/TelescopeDetector/BuildTelescopeDetector.hpp

```diff
@@ -19,22 +19,32 @@
 namespace ActsExamples {
 namespace Telescope {
 
+/// The telescope detector surface type
+enum TelescopeSurfaceType : unsigned int {
```

Comment:
Do you need the implicit convertability?
```suggestion
enum class TelescopeSurfaceType {
```

Reply:
I guess no. Fixed.